hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLog.java

```diff
@@ -88,14 +84,11 @@ void incrementCount(List<Long> txIDs)
   int resetCount(List<Long> txIDs) throws IOException;
 
   /**
-   * Commits a transaction means to delete all footprints of a transaction
-   * from the log. This method doesn't guarantee all transactions can be
-   * successfully deleted, it tolerate failures and tries best efforts to.
-   *  @param transactionResults - delete block transaction results.
-   * @param dnID - ID of datanode which acknowledges the delete block command.
+   * Get SCMDeletedBlockTransactionStatusManager.
+   * @return an Object of SCMDeletedBlockTransactionStatusManager
    */
-  void commitTransactions(List<DeleteBlockTransactionResult> transactionResults,
-      UUID dnID);
+  SCMDeletedBlockTransactionStatusManager
+      getSCMDeletedBlockTransactionStatusManager();
```

Comment:
`DeletedBlockLog` interface is defined in terms of operations .  I don't think exposing a manager object is appropriate for the interface, it should be an implementation detail.  Similarly, sharing the same lock between the two objects does not seem right.

Maybe the interface should define operations that the implementation passes through to the manager.  Alternatively the manager object should have an interface defined separately, and act as a way to manipulate the `DeletedBlockLog`.

Reply:
Removed the `getSCMDeletedBlockTransactionStatusManager` interface from `DeletedBlockLog` and added `DeletedBlockTransactionStatusManager` related actions to `DeletedBlockLog`.